### ISSUE TITLE
Sandbox didn't load within specified timeout

### ISSUE THREAD

**Issue by odinuv** [keboola/provisioning-bundle#115]
100GB table doesn't load in time https://github.com/keboola/provisioning-bundle/blob/66153d930942405a273e62f0bc71c3d384c4140c/Entity/Server/Docker.php#L701

**Comment by ondrejhlavacek** [comment]
tyvado, a chceme vůbec podporovat sandboxy, do kterejch se data loadujou dýl jak hodinu? to je musí srát tak jako tak. clone do snowflake workspace a pak tam napojit jupyter?

**Comment by odinuv** [comment]
hele nevim!

**Comment by odinuv** [comment]
ale predstavuju si, ze to jako nahodis a pak si v tom tyden neco piples

**Comment by ondrejhlavacek** [comment]
pokud na to klient nebo padák netlačí, tak bych se a to asi vybodl a zkusil to vyřešit nějak systémově jinak?

**Comment by odinuv** [comment]
predpokladam, ze HC na to tlacit bude protoze jim failuje load do sandboxu
nicmene stejne to ted nemame jak deploynout

**Comment by odinuv** [comment]
@padak mas na to nejakej nazor ?

**Comment by padak** [comment]
@odinuv 100GB tabulka v Snowflake nebo 100GB reálnejch dat? To první je nejspíš klidně 1/2 TB... 100GB chápu proč tam chtějí - potřebují s tím pracovat. Otázka je co dělají a jak jim říct že to maj dělat líp. Oni možná dělaj nějaký transformace v Pythonu, co by mělo bejt udělaný v Snowflake, etc.

Čistě "data clonem do snowflake workspace" není vždycky řešení, pokud je potřebují projít nějakou python fcí, která na Snowflake nemá ekvivalent.

**Comment by odinuv** [comment]
116.8 GB ve storage, otazka je "chceme podporovat R/Python sandboxy, do kterejch se data loadujou dýl jak hodinu?"

**Comment by padak** [comment]
Tak to je mrtě GB. Můžeš mi dát ticket nebo error job? Loopnu to s Grillem a Fíšou, jestli to dělaj "dobře". 

Podporovat bysme to měli, protože oni si nahodí prostředí ve kterém by pak chtěli dlouhodobě makat. Není to často na vyrobení transformace, ale na práci - proto se hodně ptají jak tam doloadovat data za chodu - protože je to jejich desktop a ne sandbox na budoucí transformaci.

**Comment by odinuv** [comment]
error job https://connection.keboola.com/admin/projects/5714/jobs/504443717?q=

**Comment by odinuv** [comment]
jo, prodlouzit to muzem - nic tomu technicky nebrani (krome toho, ze to nemame jak nasadit)

**Comment by padak** [comment]
@odinuv nahodil jsem DM ve slacku s holkou co tam dělá na Risku, jsi v tom.
